src/airflow_docker/operator.py

```diff
@@ -372,11 +379,11 @@ def __get_tls_config(self):
             self.docker_url = self.docker_url.replace("tcp://", "https://")
         return tls_config
 
-    def do_meta_operation(self, context, host_tmp_dir):
+    def post_execute_hook(self, context, host_tmp_dir):
```

Comment:
Dislike. The `do_meta_operation` is a part of the cooperative mulitple inheritance design.  "hooks" will, on the other hand be part of augmenting the behavior of all things with that base.

Reply:
"do_meta_operation" means absolutely nothing to me. this name sucks